python/ray/tests/modin/modin_test_utils.py

```diff
@@ -105,7 +105,7 @@ def df_equals(df1, df2):
     if isinstance(df1, pandas.DataFrame) and isinstance(df2, pandas.DataFrame):
         if (df1.empty and not df2.empty) or (df2.empty and not df1.empty):
             assert False, "One of the passed frames is empty, when other isn't"
-        elif df1.empty and df2.empty and type(df1) != type(df2):
+        elif df1.empty and df2.empty and not isinstance(df1, type(df2)):
```

Comment:
```python
elif df1.empty and df2.empty and type(df1) is not type(df2):
```

Reply:
Thanks for the feedback! I've made the changes you suggested. Let me know if anything else needs tweaking.